pkg/ccl/backupccl/restore_job.go

```diff
@@ -221,6 +223,65 @@ func makeBackupLocalityMap(
 	return backupLocalityMap, nil
 }
 
+func filterCompletedImportSpans(
+	completedSpansSlice []jobspb.RestoreProgress_RestoreProgressFrontierEntry,
+	importSpans []execinfrapb.RestoreSpanEntry,
+) (*spanUtils.Frontier, []execinfrapb.RestoreSpanEntry, error) {
+	var frontierCreationSlice []roachpb.Span
+	var modifiedImportSpans []execinfrapb.RestoreSpanEntry
```

Comment:
nit: in general, you want to declare variables as close as possible to where you first use them. I'd move the `modifiedImportSpans` declaration lower.

Reply:
further, maybe rename this to `toDoSpans`